inc/azure_c_shared_utility/umock_c_prod.h

```diff
@@ -10,6 +10,12 @@
     by adjusting the include paths).
 */
 
+#if defined(_DLL) && (defined(_WIN32) || defined(_WIN64))
```

Comment:
>_DLL [](start = 12, length = 4)

Where does the _DLL get defined?

Reply:
_DLL is automatically defined by the MS build system when the target is a dll

---
In reply to: [184152808](https://github.com/Azure/azure-c-shared-utility/pull/152#discussion_r184152808) [](ancestors = 184152808)